src/aiida/manage/configuration/migrations/migrations.py

```
@@ -391,6 +391,79 @@ def downgrade(self, config: ConfigType) -> None:
+    and ``rmq.task_timeout`` becomes ``broker.task_timeout``. Any explicitly set value is moved to the new option.
+    """
+
+    down_revision = 9
+    down_compatible = 9
+    up_revision = 10
+    up_compatible = 10
+
+    option_renames = [
+        {'v9': 'logging.db_loglevel', 'v10': 'logging.database_handler'},
+        {'v9': 'rmq.task_timeout', 'v10': 'broker.task_timeout'},
+    ]
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)